PublishWorkflow/PublishWorkflowChecker.php

```diff
@@ -98,16 +110,11 @@ public function __construct(ContainerInterface $container, AccessDecisionManager
      */
     public function getToken()
     {
-        if (null === $this->token) {
-            if ($this->container->has('security.token_storage')) {
-                return $this->container->get('security.token_storage')->getToken();
-            } elseif ($this->container->has('security.context')) {
-                // to be BC with Symfony 2.3
-                return $this->container->get('security.context')->getToken();
-            }
+        if (null !== $this->token) {
+            return $this->token;
         }
 
-        return $this->token;
+        return $this->getTokenStorage()->getToken();
```

Comment:
Imo, the bundle should come with a compiler pass and replace the publish workflow checker service with the AlwaysPublishedWorkflowChecker when security is not available.